raceplotly/plots.py

```diff
@@ -289,5 +290,8 @@ def __define_ui(self, time_label: str = None):
         
 
     def __get_colors(self):
-        colors = {item: 'rgb({}, {}, {})'.format(*sample(range(256), 3)) for item in self.df[self.item_column].unique()}
-        self.df['color'] = self.df[self.item_column].map(colors)
+        if (self.item_color == None):
+            colors = {item: 'rgb({}, {}, {})'.format(*sample(range(256), 3)) for item in self.df[self.item_column].unique()}
+            self.df['color'] = self.df[self.item_column].map(colors)
+        else:
```

Comment:
The only comment about this approach I have is that the users needs to either enter all the colours manually or have them all be random, another option could be to have certain colours specified in a `Dict` as you've done it and to have the remaining be assigned randomly. Though this is not a deal braker, if in the future a user wishes for that functionality to be added we can add it

Reply:
Yeah, you are right. I was thinking about the same while working on it. And I found out that if we don't mention a specific color to a category then by default it assigns Black color to itself. So for example, if we have 10 categories and I have explicitly mentioned colors to 6 of them then the rest four will be black. 

But yeah I will try doing it. It would be great if instead of black the remaining 4 categories get random colors.

Reply:
Hey no worries, if you choose to work on that feature in the future, make sure to create a new pull request for that given that this one has already been merged!